atomic_reactor/pulp_util.py

```diff
@@ -101,7 +101,7 @@ def _set_auth(self):
             # Tell dockpulp.
             self.p.set_certs(cer, key)
 
-    def _create_missing_repos(self, pulp_repos, repo_prefix):
+    def _ensure_repos(self, pulp_repos, repo_prefix):
         repos = pulp_repos.keys()
         found_repos = self.p.getRepos(repos, fields=["id"])
```

Comment:
We should use this opportunity to figure out whether we need to update auto_publish.

Reply:
Let's also make sure we keep backwards compatibility. For instance, if the version of dockpulp doesn't support checking repo's `auto_publish` status, there should be a fallback. This fallback could be be simply to update repo anyways, but... support for changing `auto_publish` is also not in the dockpulp version. So the fallback should be a noop. Thoughts?

Reply:
Fallback should definitely be a no-op.